test/bin/manage_brew_rpms.sh

```diff
@@ -38,10 +42,15 @@ action_download() {
 
     # Attempt downloading the specified build version
     local package
-    package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}" | sort | tail -1) || true
+    case ${ver_type} in 
+        zstream) package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}" | grep -v "~" | uniq | tail -n1) || true;;
+        rc|ec)   package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}.0~${ver_type}." | tail -1) || true;;
+        *)       package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}.*${ver_type}" | tail -1) || true;;
+    esac
```

Comment:
Nit: we usually use this type of formatting, which is a bit more readable.
```suggestion
    case ${ver_type} in 
    zstream)
        package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}" | grep -v "~" | uniq | tail -n1) || true
        ;;
    rc|ec)
        package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}.0~${ver_type}." | tail -1) || true
        ;;
    *)
    package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}.*${ver_type}" | tail -1) || true
    ;;
    esac
```